src/main/resources/assets/computercraft/lua/rom/programs/http/webrun

```diff
@@ -0,0 +1,25 @@
+--Made by Wilma456
+local args = {...}
+
+if not args[1] then
+  print("Usage: webrun <url> <args>")
+  return
+end
+
+if not http then
+  print("Please enable http in the config")
+  return
+end
+
+local test,text = http.checkURL(args[1])
+if test == false then
+  print("[Error]"..text)
+  return
+end
+
+local data = http.get(args[1])
+local program = data.readAll()
+data.close()
+local run = load(program)
```

Comment:
The code source should be specified, either as the URL or the filename extracted from the URL. `load(program, source)`

Additionally, syntax errors will cause `run` to be `nil` and a second value containing the error message to be returned. This should be handled gracefully via `local run, err = load(...) if not run ... else ... end`